.github/workflows/knative-style.yaml

```diff
@@ -41,13 +41,13 @@ jobs:
 
     steps:
       - name: Set up Go
-        uses: actions/setup-go@v2
```

Comment:
Ref: https://stackoverflow.com/questions/71758856/github-action-for-golangci-lint-fails-with-cant-load-fmt